externals/autotools/share/autoconf/autom4te.cfg

```diff
@@ -106,7 +106,7 @@ end-language: "Autoreconf-preselections"
 # This intermediate language is used by aclocal to build aclocal.m4.
```

Comment:
I don't think it is necessary to modify the externals/ directory for these updates, can you please remove these files from the pull request?